app/views/pages/accessibility_statement.html.erb

```diff
@@ -0,0 +1,182 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">
+      Accessibility statement for Teachers: claim back your student loan repayments
+    </h1>
+
+    <p class="govuk-body">
```

Comment:
Is this missing an `l` level heading?

Reply:
Perhaps, its based off the sample page which I don't think had one. Can amend if it make it easier to read.